### DISCUSSION TITLE
Feature: Shinobi CCTV/NVR in store

### DISCUSSION THREAD

**Discussion by Protektor-Desura** [IceWhaleTech/CasaOS#237] [Ideas]
I thought I would recommend another app/docker you should think about adding/supporting since you have talked about supporting home cameras.

Shinobi Open-Source CCTV/NVR
https://shinobi.video/
https://hub.docker.com/r/shinobicctv/shinobi/
https://gitlab.com/Shinobi-Systems/Shinobi

It has support for object detection, motion detection, ALPR (automatic license plate reader), and other options.

**Comment by SmarterPapa** [comment]
are there any news on the topic home camera video recognition software for CasaOS?
